endpoints/globus/endpoint.go

```diff
@@ -68,8 +68,11 @@ type Endpoint struct {
 }
 
 func NewEndpoint(endpointName string) (core.Endpoint, error) {
-	epConfig, found := config.Globus.Endpoints[endpointName]
+	epConfig, found := config.Endpoints[endpointName]
 	if !found {
+		return nil, fmt.Errorf("'%s' is not an endpoint", endpointName)
+	}
+	if epConfig.Provider != "globus" {
```

Comment:
are there tests for these two errors? (I love me some comprehensive error testing!)

Reply:
There are now! :-)